Sources/Plasma/CoreLib/plFileSystem.cpp

```diff
@@ -64,6 +64,7 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #   if defined(__has_include)
 #     if __has_include(<sysdir.h>)
 #       include <sysdir.h>
+#       define HAS_SYSDIR
```

Comment:
CoreLib is using unity build, so this define may leak out into other TUs. It might be better to move the sysdir detection out to CMake or test for if `__builtin_available()` functions in CMake.

Reply:
Testing for `__builtin_available()` with CMake is probably the right approach